release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -178,17 +178,42 @@ Limit Calculator] to estimate cluster limits for your environment.
 [id="ocp-4-3-storage"]
 === Storage
 
-[id="ocp-4-3-iscsi"]
-==== iSCSI
+[id="ocp-4-3-ocs-4-2"]
+==== OpenShift Container Storage 4.2
 
-Persistent volumes and raw block volumes using iSCSI, previously in Technology
-Preview, are now fully supported in {product-title} 4.3.
+Deploy, manage, monitor, and migrate a Red Hat OpenShift Container Storage 4.2 cluster. See the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.2/html/4.2_release_notes/[Red Hat OpenShift Container Storage 4.2 Release Notes].
+
+[id="ocp-4-3-iscsi-persistent-storage"]
+==== Persistent storage Using iSCSI
+
+Persistent volumes using iSCSI, previously in Technology Preview, is now fully supported in {product-title} {product-version}.
+
+[id="ocp-4-3-raw-block-volume"]
+==== Raw block volume support
+
+iSCSI raw block volumes are now fully supported with OpenShift Container Platform 4.3.
+
+Raw block volumes using Cinder are now in Technology Preview.
 
 [id="ocp-4-3-csi-volume-expansion"]
-==== Container Storage Interface
+==== CSI volume expansion
+
+You can now use the Container Storage Interface (CSI) to expand storage volumes after they have already been created. This feature is enabled by default in Technology Preview.
+
+[id="ocp-4-3-lso-tolerations"]
+==== Use tolerations in Local Storage Operator
+
+The Local Storage Operator now tolerates node taints, allowing you to provision local volumes from tainted nodes.
 
-Using the Container Storage Interface (CSI) to expand storage volumes after they
-have already been created is now enabled by default in Technology Preview.
+[id="ocp-4-3-local-provisioner"]
+==== Local storage provisioner removed
+
+The previously deprecated technical preview of the `ose-local-storage-provisioner` container has been removed. It is recommended to use the OLM-based Local Storage Operator (`ose-local-storage-operator`).
```

Comment:
"technical preview" should be technology, and capitalized: `The previously deprecated Technology Preview`